bin/install

```diff
@@ -46,8 +46,22 @@ get_binary_path_in_archive() {
   echo "$archive_dir/$binary_name"
 }
 
+get_arch() {
+  local arch; arch=$(uname -m | tr '[:upper:]' '[:lower:]')
+  case ${arch} in
+  x86_64)
+    arch='x86_64'
+    ;;
+  arm64)
+    arch='arm64'
+    ;;
+  esac
+
+  echo "${arch}"
+}
+
 get_platform() {
-  echo "$(uname)_x86_64"
+  echo "$(uname)_$(get_arch)"
```

Comment:
Bad juju here! Platform and architecture should not mix until the download URL is built. Platform is really the OS and architecture is the CPU. By mixing them here one is just setting themselves up for a lot of heartache when the platform needs to be aliased. Take a look at https://gitlab.com/wt0f/asdf-fx/-/blob/main/bin/install?ref_type=heads#L63-72